packages/@office-iss/react-native-win32-tester/overrides.json

```diff
@@ -27,11 +27,8 @@
       "baseHash": "20e35122c8529a5bf848f91d9ea9b85be3c23882"
     },
     {
-      "type": "copy",
-      "file": "src/js/RNTesterApp.win32.js",
-      "baseFile": "packages/rn-tester/js/RNTesterApp.android.js",
-      "baseHash": "30f3ef9bdc0e2c337309079831664c4a596c404f",
-      "issue": 4586
```

Comment:
We want to keep this updated when changes come in. Could you revert this?

If there are changes to app behavior itself, I thin you're looking for RNTesterAppShared. RNTesterApp in newer versions is just a light shim.

Reply:
I'll remove the change; I like having the messagequeue comments in place to find them instead of determining where to put them, but it's not necessary